### PR TITLE
Add from_str helper to IpNetwork

### PR DESCRIPTION
**Pull request by cholcombe973**
I added a simple helper function that I found useful in another project.

### CODE REVIEW

**Comment by achanda** [comment]
Looks like this would be more idiomatic as `impl FromStr for IpNetwork`, no?

File: src/lib.rs [hunk 1]
```diff
@@ -35,6 +35,33 @@ impl IpNetwork {
...
+    /// network size.  If the prefix is larger than 32 (for IPv4) or 128 (for IPv6), this
+    /// will raise an `IpNetworkError::InvalidPrefix` error.
+    /// # Examples
+    ///
+    /// ```
+    /// use std::net::Ipv4Addr;
+    /// use ipnetwork::Ipv4Network;
+    /// use ipnetwork::IpNetwork;
+    ///
+    /// let new = IpNetwork::V4(Ipv4Network::new(Ipv4Addr::new(10, 1, 9, 32), 16).unwrap());
```
Comment (achanda) [inline src/lib.rs]: Let's use a different name for the var here

**Comment by cholcombe973** [comment]
Yeah I thought about that but you can't have an impl on an enum :(

**Comment by cholcombe973** [comment]
Yeah definitely.  I moved it into an impl FromStr

File: src/lib.rs [hunk 2]
```diff
@@ -35,6 +37,8 @@ impl IpNetwork {
         }
     }
 
+
```
Comment (achanda) [inline src/lib.rs]: Let's remove those extra empty lines

**Comment by achanda** [comment]
Thanks! Looks good other than the nit about empty lines.

**Comment by cholcombe973** [comment]
Thanks :)

**Comment by achanda** [comment]
@cholcombe973 you might want to open a PR for `IpAddr` in the rust project. I think this will be a nice addition there.

**Comment by cholcombe973** [comment]
Cool I'll go for it.  I've wanted to become a contributor for awhile now :)

**Comment by cholcombe973** [comment]
@achanda are you talking about: https://doc.rust-lang.org/src/std/up/src/libstd/net/parser.rs.html#306-314 ?

**Comment by achanda** [comment]
Ah, looks like it's already there!

**Pull request merged**